appinfo/info.xml

```diff
@@ -23,6 +23,9 @@ While theoretically any other authentication provider implementing either one of
 	<types>
 		<authentication/>
 	</types>
+	<documentation>
+		<admin>https://portal.nextcloud.com/article/configuring-single-sign-on-10.html</admin>
```

Comment:
Is there no unique direct link?
I think we should not use text like that that can potentially change? :thinking:

Reply:
Not that I'm aware of. What do you suggest instead?

Reply:
Nothing if there is no alternative :p